app/views/casework/crime_applications/_case_details.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= label_text(:case_details) %>
 </h2>
 <% if case_details.case_type.present? %>
-  <dl class="govuk-summary-list govuk-!-margin-bottom-9">
```

Comment:
Why was this removed?

Reply:
Good Catch 👍 
I added 1 more <dl> tag and last <dl> should have `govuk-!-margin-bottom-9` class, but it was conditional.
Here is the conditional structure now. The new attributes should be nested under `<% if case_details.case_type.present? %>` condition
<img width="610" alt="Screenshot 2024-02-02 at 12 10 47" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/195928/e99e7783-6998-41db-acfb-1b3c40b66465">